recipes/ntp/ntp.inc

```diff
@@ -77,7 +77,7 @@ DEFAULT_USE_ntpd_sysvinit_stop	= "25"
 
 inherit auto-package-utils
 AUTO_PACKAGE_UTILS = "ntpd ntpdate ntp-wait ntp-keygen sntp ntpq ntpdc \
-    ntptime ntptrace tickadj"
+    ntptime ntpsweep ntptrace tickadj calc_tickadj update-leap"
```

Comment:
Just curious: I see the ntpsweep perl script in the build/ dir, but it doesn't seem to get installed. Does it need some special configure option, or does it magically detect that the target doesn't have perl, or?